### PR TITLE
docs: Add sponsors to README

### PR DESCRIPTION
**Pull request by mdjermanovic**
Adds sponsors to README.md. I added a script and a workflow that automatically updates sponsors every day.

### CODE REVIEW

File: tools/update-readme.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ */
+async function fetchSponsorsMarkdown() {
+	return got(SPONSORS_URL).text();
+}
+
+//-----------------------------------------------------------------------------
+// Main
+//-----------------------------------------------------------------------------
+
+(async () => {
```
Comment (fasttime) [inline tools/update-readme.js:41]: Is this IIFE actually needed?

Comment (mdjermanovic) [reply tools/update-readme.js]: I've removed it now.

**Review by amareshsm — APPROVED** [review]
LGTM

**Review by fasttime — APPROVED** [review]
LGTM, thanks!

**Pull request merged**